extensions/amp-accordion/1.0/accordion.js

```diff
@@ -375,7 +397,7 @@ export function AccordionHeader({
       className={`${className} ${classes.sectionChild} ${classes.header}`}
       tabIndex={tabIndex}
       aria-controls={contentId}
-      onClick={expandHandler}
+      onClick={() => toggleHandler()}
```

Comment:
nit: 
```suggestion
      onClick={toggleHandler}
```

Reply:
The first argument `toggleHandler` accepts is the `opt_expand` (force expand or force collapse).  Wrapping the onClick callback in this function prevents the `click` event from being passed into `toggleHandler` as the first argument.